knowledge-base/wizard-confirm-step-change.md

```diff
@@ -0,0 +1,83 @@
+---
+title: How do I pop up a confirmation window or dialog between Wizard steps?
+description: How to show a dialog confirming the change of the wizard steps
+type: how-to
+page_title: Confirm between Wizard steps
+slug: wizard-kb-confirm-step-change
+position: 
+tags: wizard
+ticketid: 1559188
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Wizard for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+When a user wants to move to the next step, I need to confirm the move (e.g. show them a warning for some situations), and if they choose to "continue" take them to the next step as normal, not back to the current step.
```

Comment:
```suggestion
When a user wants to move to the next step, I need to confirm the move and show them a popup warning for some situations. If they choose to "continue", take them to the next step as normal, otherwise back to the current step.
```